app/src/main/java/org/kiwix/kiwixmobile/utils/KiwixDialog.kt

```diff
@@ -16,6 +16,28 @@ sealed class KiwixDialog(
     constructor(bookOnDisk: BookOnDisk) : this(arrayOf(bookOnDisk.book.title))
   }
 
+  object LocationPermissionRationale : KiwixDialog( // For the local file transfer module
```

Comment:
The comment is unnecessary and I believe the text is not changed from how this dialog would normally show? So the comment is also inaccurate. Same for the comment below

Reply:
I'll remove the comments
> I believe the text is not changed from how this dialog would normally show

Didn't get this part

Reply:
So we (should) show this dialog whenever we ask for LocationPermissions, it does not matter that it is used in local file transfer, it can be used anywhere. It is the same text regardless of what screen it is shown in